object_detection/g3doc/preparing_inputs.md

```diff
@@ -8,38 +8,45 @@ TFRecords.
 ## Generating the PASCAL VOC TFRecord files.
 
 The raw 2012 PASCAL VOC data set can be downloaded
-[here](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar).
+[here](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar),
+or by using the command below.
```

Comment:
This should also apply to the extraction step. Perhaps we can reword it like so:

"The raw 2012 PASCAL VOC dataset is located [here](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar). To download, extract and convert it to TFRecords, run the following commands below: